keptn/keptn
# feat(installer): Add options for setting image repository and tag globally

## This PR treats with #7578 and is based on PR #7695

### helm-service + jmeter-service + installer/manifests/keptn
- introduce set values for image registry and repo globally in values.yaml
- extend common with _images.tpl

### installer/manifests/common
- crated `_images.tpl` and implemented functions `common.images.image` and `common.images.tag`

### How to test
Example for `installer/manifests/keptn`
- Checkout master 
`helm template . --name-template test-helm-keptn --output-dir ../../../../tmp/keptn-chart-test/tempMaster`
- Checkout PR
`helm template . --name-template test-helm-keptn --output-dir ../../../../tmp/keptn-chart-test/tempPR`
- Compare now templated Manifests
- Test local installation in cluster with custom changes
see `docs/developer/install_local_customisations.md`

Fixes #7578 
Fixes #6043
Improves #7660
